docs/design.md

```diff
@@ -469,7 +469,7 @@ Please see [pkg/roachpb/data.proto](https://github.com/cockroachdb/cockroach/blo
 
 **Choosing a Timestamp**
 
-A key challenge of reading data in a distributed system with clock skew
+A key challenge of reading data in a distributed system with clock offset
```

Comment:
I think this line was ok with the word "skew", since it's the fact that the machines have clock skew that causes an offset between their clocks.